examples/e2e-tests/kubernetes/kubernetes-config/kata-containers.json

```diff
@@ -0,0 +1,40 @@
+{
+  "apiVersion": "vlabs",
```

Comment:
I don't think we need the example json in both places, we have a //todo item to move the feature examples in `e2e-tests/` so you can probably keep just the other one

Reply:
ACK.